server/routes/availability.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+const {
+ createSchedule,
+ getSchedule,
+ getActiveSchedule, 
+ getAllSchedules, 
+ makeActiveSchedule
+} = require('../controllers/availability');
+
+router.route('/availability').post(createSchedule);
```

Comment:
```
router.route('/').post(createSchedule);
router.route('/active').get(getActiveSchedule)
router.route('/:scheduleId').get(getSchedule)
router.route('/all').get(getAllSchedules)
router.route('/:scheduleId/activate').patch(makeActiveSchedule)
```
The HTTP verbs indicate the purpose of the route, we don't need to specify explicit route names unless necessary